packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/CameraProperties.java

```diff
@@ -147,13 +147,35 @@ public interface CameraProperties {
    * height and crop region height, for @see android.scaler.cropRegion.
    *
    * <p>By default maps to the @see
-   * android.hardware.camera2.CameraCharacteristics#SCALER_AVAILABLE_MAX_DIGITAL_ZOOM key.
+   * android.hardware.camera2.CameraCharacteristics#SCALER_AVAILABLE_MAX_DIGITAL_ZOOM key
```

Comment:
Why remove the periods here and below? I think everything should be made consistent and looks like most things end in periods?